services/core-web/src/components/dashboard/noticeOfWorkHomePage/NoticeOfWorkHomePage.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { useState, useEffect } from "react";
+import { useLocation, useHistory } from "react-router-dom";
+import { Action, bindActionCreators, Dispatch } from "redux";
+import { connect } from "react-redux";
+import queryString from "query-string";
+import * as Strings from "@common/constants/strings";
+import {
+  getMineRegionHash,
+  getMineRegionDropdownOptions,
+  getDropdownNoticeOfWorkApplicationStatusOptions,
+  getDropdownNoticeOfWorkApplicationTypeOptions,
+} from "@common/selectors/staticContentSelectors";
+import { fetchNoticeOfWorkApplications } from "@common/actionCreators/noticeOfWorkActionCreator";
+import {
+  getNoticeOfWorkList,
+  getNoticeOfWorkPageData,
+} from "@common/selectors/noticeOfWorkSelectors";
+import * as routes from "@/constants/routes";
+import NoticeOfWorkTable from "@/components/dashboard/noticeOfWorkHomePage/NoticeOfWorkTable";
+import NoticeOfWorkSearch from "@/components/dashboard/noticeOfWorkHomePage/NoticeOfWorkSearch";
+import ResponsivePagination from "@/components/common/ResponsivePagination";
+import { PageTracker } from "@common/utils/trackers";
+import { INoticeOfWorkApplication, IPageData, IOption } from "@mds/common";
+import { RootState } from "@/App";
+
+export interface NoWSearchParams {
```

Comment:
Will this interface be used in other pages ?

Reply:
I'm using it in both the HomePage and the Table file, but I don't _expect_ to use it anywhere else. Wasn't sure where the best place to put it was.

Reply:
I was wondering if core-web/common would be a good place to put it.

Reply:
My thinking is that the interface is specifically for search parameters for this page, and so it belongs with the page. I think it would be clutter/obsfucation in core-web/common. ie: within the common directory, where would it make sense? Make a directory /pages/searchParams? Would we be moving it to the common directory just because it's an interface (and not for props) but not because it's needed by both core-web and minespace-web?

Reply:
Sounds good. Maybe pages/searchParams can work, but I am also okay with leaving it as is since it is used by only 2 components.

Reply:
I think I prefer leaving it where it is for this case.  That, or maybe create it's own interface file within this same folder?

core-web/common will eventually be deleted hopefully, and this isn't an interface that is needed in Minespace, so I think placing it there wouldn't be the best fit.  Could maybe put it in src/interfaces for core-web.